BTrees/_base.py

```diff
@@ -1066,6 +1076,10 @@ def _p_resolveConflict(self, old, com, new):
         return ((
             self._bucket_type()._p_resolveConflict(s_old, s_com, s_new), ), )
 
+    def __repr__(self):
+        r = super(_Tree, self).__repr__()
+        r = r.replace('Py', '')
```

Comment:
L162 is in `_BucketBase`, which is used by `Bucket` and `Set`. This line is in `_Tree`, used by `TreeSet` and `BTree`.